Content.Shared/Movement/Systems/SharedMoverController.cs

```diff
@@ -285,6 +289,13 @@ public override void UpdateAfterSolve(bool prediction, float frameTime)
             PhysicsSystem.SetAngularVelocity(physicsUid, 0, body: physicsComponent);
         }
 
+        public void WalkingAlert(EntityUid player, bool walking) {
+            if (_tags.HasTag(player, "CanWalk"))
+            {
+                _alerts.ShowAlert(player, AlertType.Walking, walking ? (short) 0 : (short) 1);
+            }
+        }
+
```

Comment:
I'm revising my review. Please change this to use a CanWalkComponent, which will naturally require that such a component be made and added to the BaseSpecies and BaseAnimal entities. This will prevent all future conflicts related to the TagSystem being awful garbage. 
```suggestion
        public void WalkingAlert(EntityUid player, bool walking) {
            if (HasComp<CanWalkComponent>(player))
            {
                _alerts.ShowAlert(player, AlertType.Walking, walking ? (short) 0 : (short) 1);
            }
        }

```